context/app/static/js/components/detailPage/visualization/VisualizationWorkspaceButton/VisualizationWorkspaceButton.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import SvgIcon from '@mui/icons-material/Download';
+
+import { WorkspacesIcon } from 'js/shared-styles/icons';
+import NewWorkspaceDialog from 'js/components/workspaces/NewWorkspaceDialog';
+import { useCreateWorkspaceForm } from 'js/components/workspaces/NewWorkspaceDialog/useCreateWorkspaceForm';
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
+import { WhiteBackgroundIconButton } from 'js/shared-styles/buttons';
+import { useAppContext } from 'js/components/Contexts';
+
+const tooltip = 'Launch New Workspace';
+
+interface VisualizationWorkspaceButtonProps {
+  uuid?: string;
+  hubmap_id?: string;
+  hasNotebook?: boolean;
+  mapped_data_access_level?: string;
+}
+
+function VisualizationWorkspaceButton({
+  uuid = '',
+  hubmap_id,
+  hasNotebook,
+  mapped_data_access_level,
+}: VisualizationWorkspaceButtonProps) {
+  const { isWorkspacesUser } = useAppContext();
+  const { setDialogIsOpen, removeDatasets, ...rest } = useCreateWorkspaceForm({
+    defaultName: hubmap_id,
+    defaultTemplate: 'visualization',
+    initialSelectedDatasets: [uuid],
+  });
+
+  if (!isWorkspacesUser ?? !uuid ?? !hubmap_id ?? !hasNotebook ?? mapped_data_access_level === 'Protected') {
+    return null;
+  }
```

Comment:
Do we need all of this? Would `!isWorkspaceUser || !hasNotebook` suffice? In which cases are `uuid` and `hubmap_id` undefined?

Reply:
@austenem I don't believe there are any cases where `hasNotebook && mapped_data_access_level === Protected` can we drop the check for `mapped_data_access_level`?